app/code/core/Mage/Adminhtml/etc/config.xml

```diff
@@ -236,23 +236,10 @@
         </help>
         <admin>
             <design>
```

Comment:
`<design>` should be `<theme>` . See [system,xml](https://github.com/OpenMage/magento-lts/blob/916adc7a46c704628c61ce793ed1cde24b649be8/app/code/core/Mage/Core/etc/system.xml#L1061).